ocean_lib/assets/test/test_asset.py

```diff
@@ -49,11 +49,13 @@ def test_trusted_algorithms(publisher_ocean_instance):
 
     trusted_algorithms = ddo.get_trusted_algorithms()
     service = ddo.get_service(ServiceTypes.CLOUD_COMPUTE)
-    privacy_dict = service.attributes["main"].get("privacy", {})
+    privacy_dict = service.attributes["main"].get("privacy")
+    if not privacy_dict:
```

Comment:
I think returning `None` from the test will cause the rest of the assert statements to be skipped. Please replace this if with `assert privacy_dict`

Reply:
I agree. I missed this, but at this point there should be a privacy dict there. Yes, please follow David's suggestion.

Reply:
Done

Reply:
@mariacarmina I'm afraid you misunderstood me.  

Please make it look like the following.  Remove the `if` and `return` statements.

```python
privacy_dict = service.attributes["main"].get("privacy")
assert privacy_dict
```